styles.css

```diff
@@ -18,6 +18,33 @@ p {
     background-color: #1BAB49;
 }
 
+.ingredient {
+    border-radius: 15px;
+    box-shadow: 0 3px 15px rgba(0,0,0,0.2);
```

Comment:
Also, I'd suggest the following for the box shadow:
`box-shadow: 2px 2px 3px rgba(30,30,30,0.4);`